LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2014 Fabien Potencier
+Copyright (c) 2019 Derek Stephen McLean
```

Comment:
you may not modify the license. even if you fork

Reply:
its not actually a fork anymore, all the pimple code is gone and pimple is now simply a requirement in my package, but i dont know how to remove the upstream on github. maybe i should just create a fresh repository?

Reply:
- https://www.google.com/search?q=github+remove+fork+relation
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/detaching-a-fork

Reply:
and additionally you can contact github support to remove fork relation if you don't want to re-create repository, because want to preserve issues and pull requests and their discussions.

Reply:
thanks for the help!